mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -303,3 +308,24 @@ def verify_flash_success(self, new_target, target, tail):
 
         self.logger.debug("ready")
         return EXIT_CODE_SUCCESS
+
+    def _check_mount_point_disappear(self, mount_point):
+        # check if mount point disappear after file copied into device
+        mount_point_previous_mounted = None
+
+        mount_point_disappear_timeout = 20
+
+        while mount_point_disappear_timeout > 0:
+            if os.path.ismount(mount_point):
+                mount_point_previous_mounted = True
+
+            if not os.path.ismount(mount_point) and mount_point_previous_mounted is True:
+                self.logger.info("mount point disappears now after file copied")
+                return EXIT_CODE_SUCCESS
+
+            sleep(1)
+            mount_point_disappear_timeout -= 1
+
+        self.logger.error("After file copied, mount point did not disappear in the system in %i seconds",
+                          mount_point_disappear_timeout)
+        return EXIT_CODE_MOUNT_POINT_DISAPPEAR_TIMEOUT
```

Comment:
maybe raising exception would be better here - exit codes should be used only when really exit app - to avoid confusions..